src/components/UI/Input/Input.tsx

```diff
@@ -4,7 +4,11 @@ import tw from "twin.macro";
 
 const InputStyled = styled.input`
   &::placeholder {
-    ${tw`text-gray-500 text-base`}
+    ${tw`text-cloud-300 text-base`}
+  }
+
+  &:focus {
+    ${tw`border-cloud-900 outline-none`}
   }
```

Comment:
could shift this over to input tag: `focus:border-cloud-900 focus:outline-none`

Reply:
done.